mqtt.kafka.broker/zilla.yaml

```diff
@@ -57,9 +112,10 @@ bindings:
       bootstrap:
         - mqtt-messages
         - mqtt-retained
+        - mqtt-places
```

Comment:
Should `mqtt-devices` in routes above via `with`, and this `mqtt-places` be the same `kafka` topic?